packages/shared/fastfetchbot_shared/services/scrapers/__init__.py

```
@@ -0,0 +1,31 @@
+    "twitter",
+    "weibo",
+    "bluesky",
+    "reddit",
+    "xiaohongshu",
+    "zhihu",
+    "douban",
+    "instagram",
+    "threads",
+    "wechat",
+    "general",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)